lib/sync/index.js

```diff
@@ -35,25 +36,22 @@ function connect(mongoDBConnectionUrl, mongoDBConnectionOption, redisUrl, cb) {
       MongoClient.connect(mongoDBConnectionUrl, mongoDBConnectionOption || {}, callback);
     },
     function connectToRedis(callback) {
-      if (redisUrl) {
-        var redisOpts = {
-          url: redisUrl
-        };
-        var client = redis.createClient(redisOpts);
-        // We don't want the app to crash if Redis is not available.
-        client.on('error', handleRedisError);
-        return callback(null, client);
-      } else {
-        return callback();
-      }
+      var redisOpts = {
+        url: redisUrl
+      };
+      var client = redis.createClient(redisOpts);
+      // We don't want the app to crash if Redis is not available.
+      client.on('error', handleRedisError);
+      return callback(null, client);
     }
   ], function(err, results) {
-    if (err) return cb(err);
+    if (err) throw err;
```

Comment:
just a small thing, if keep it consistently, let's add `{}` around the `throw` statement.